src/chicken-parmigiana.md

```diff
@@ -0,0 +1,54 @@
+# Chicken Parmigiana
+
+Staple Australian pub food (aka Chicken Parma).
+
+![Chicken Parmigiana](pix/chicken-parmigiana.webp)
+
+- Prep time: 10 min
+- Cook time: 30 min
+- Servings: 2
+
+## Ingredients
+
+- 1 boneless/skinless chicken breast
+- 1 cup plain flour
+- 2 eggs
+- 1 cup breadcumbs
+- 1/2 cup grated parmesan cheese
+- italian mixed herbs
+- several tablespoons olive oil
+- 4 TBsp parmigiana sauce
+- 200g sliced mozzarella
+- 100g sliced ham off the bone
+
+## Directions
+
+1. Preheat oven to ~180 celcius.
+2. Slice chicken thinly into strips of roughly 2-3cm. This depends entirely on the size of the chicken breast, most breasts should make 2 or 3 good strips.
+3. Prepare several dipping stations: One bowl of flour, one of beaten eggs, one of breadcrumbs + parmesan cheese + sprinkle of italian mixed herb (to taste).
+4. Coat frying pan in olive oil and bring to a medium/high heat.
+5. Coat chicken strips completely in flour to assist the breading process.
+6. Dip the floured chicken strips in the egg wash.
+7. Coat egg washed chicken strips in bread crumbs.
+8. Fry breaded chicken roughly 7 minutes either side, ensure breading is golden brown.
+9. Transfer breaded chicken to oven pan.
+10. Top the breaded chicken with a slice of ham, followed by the parmigiana sauce, then a slice of mozzarella.
+11. Bake until the cheese is melted and golden (~10 minutes).
+12. Serve alone or with a side (salad/potato chips/vegetables etc).
+
+## Note
+
+Once the initial breaded chicken is cooked, you can delay the final cooking (with topping) for several hours if need be (just keep it in the fridge), to allow preparation beforehand.
+
+Multiple variations can be made depending on available ingredients (this is pub grub, use what you have to make it work):
+- bread crumbs- can use panko, crushed corn flakes etc
+- italian herbs can be added at any stage (i.e. basil leaf on top of mozzarella)
+- mozzarella can be grated, you can also just use any grated cheese if needed
+- parmigiana sauce- if you're feeling fancy, you can make this yourself with tomato passata/onion/garlic/broth/herbs etc- ideally use a pre-made variant for simplicity
```

Comment:
Indentation/formatting could be improved here.

Reply:
Think I've made the formatting look tidier here in latest update